aten/src/ATen/native/miopen/Conv_miopen.cpp

```diff
@@ -669,6 +669,10 @@ Tensor miopen_convolution_forward(
                                      padding, stride, dilation, groups),
                     input->options());
 
+  if (output_t.numel() == 0) {
+    return output_t;
```

Comment:
Did the necessity of making backend-level changes arise due to the implementation being scattered between TH and ATen? I'm working on `ConvTranspose` for 0-dim tensors and all it  is in ATen, which I think would be helpful in simply working with the `_convolution` function for the forward pass and leaving the rest untouched.